src/main/java/frc/robot/commands/GetNoteX.java

```diff
@@ -89,15 +91,10 @@ public GetNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVision note
        addCommands(new PrintCommand("GetNoteX-- Finished target note: "+ targetNote));
     }
 
-    public Pose2d getInitialPose() {
-        PathPlannerPath firstPath = m_candidateStartPaths.values().iterator().next();
-        return FieldConstants.flipPose(firstPath.getStartingDifferentialPose());
-    };
-
     private PathPlannerPath getInitialPath() {
         Pose2d pose = m_driveTrain.getPose();
         Pose2d poseBlue = FieldConstants.flipPose(pose);
-
+        System.out.println("Starting getInitialPath " + poseBlue);
         // this part used when in center note area, if intended center note is not found
         if (poseBlue.getX() > FieldConstants.BLUE_WING_LINE_X_METERS) {
```

Comment:
Doesn't this value have to get flipped if we're Red?

Reply:
"pose" is the current real pose. If we are Red, the flipPose call will flip it, and not if we are Blue, so "poseBlue" is on the Blue side. Then we can just compare with Blue landmarks.